packages/botonic-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@botonic/core",
-  "version": "0.16.0",
+  "version": "0.16.0-alpha.7",
```

Comment:
Shouldn't be the version `0.16.1-alpha.7` instead of `0.16.0-alpha.7`? The `0.16.0` is already the latest version published for `@botonic/core`.

Reply:
yes, somehow sb didn't updated it to 0.16.1, will change it in next alphas